docs/integration_quality_scale_index.md

```diff
@@ -36,12 +36,14 @@ This is a solid integration that is able to survive poor conditions and can be c
 - Satisfying all Silver level requirements.
 - Configurable via config entries.
   - Don't allow configuring already configured device/service (example: no 2 entries for same hub)
-  - Tests for the config flow
   - Discoverable (if available)
   - Set unique ID in config flow (if available)
   - Raise `ConfigEntryNotReady` if unable to connect during entry setup (if appropriate)
 - Entities have device info (if available) ([docs](device_registry_index.md#defining-devices))
-- Tests for fetching data from the integration and controlling it ([docs](development_testing.md))
+- Tests
+  - Full test coverage for the config flow
```

Comment:
This is required for all config_flows regardless of quality scale?

Reply:
Right, that might be another error. This line was simply moved a bit from above.

Reply:
Yes, but if I am not wrong, some of the ones that are already merged are missing coverage. I did not add this as a new requirement, just moved it to the "Tests" section.